my-app/src/Meals.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react'
+import "./Meals.css"
+
+function Meals() {
+  const [meals, setMeals] = useState([])
+
+    useEffect(()=>{
+        fetch("http://localhost:5000/api/meals")
+          .then((response)=> response.json())
+          .then((data)=>setMeals(data))
+    },[meals])
```

Comment:
You don't really need any dependency here in this effect :) you can remove it to avoid it re-rendering
```suggestion
    },[])
```